pandas/io/parquet.py

```diff
@@ -18,20 +18,23 @@ def get_engine(engine: str) -> "BaseImpl":
 
     if engine == "auto":
         # try engines in this order
-        try:
-            return PyArrowImpl()
-        except ImportError:
-            pass
+        engine_impls = [PyArrowImpl, FastParquetImpl]
 
-        try:
-            return FastParquetImpl()
-        except ImportError:
-            pass
+        error_msgs = []
+        for eimpl in engine_impls:
+            try:
+                return eimpl()
+            except ImportError as ie:
+                error_msgs.append(ie.msg)
```

Comment:
mypy is complaining because of this, not sure is `str(ie)` would be the correct way.

Reply:
Yup, saw that in CI, now fixed!